src/index.html

```diff
@@ -7,8 +7,79 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div>
+        <a href="#home">
+          <img
+            src="images/logo.png"
+            alt="moyo logo"
+            class="image"
+          >
+        </a>
+      </div>
```

Comment:
I'm not sure why you added this wrapper for the link. But looks like it's unnecessary 

```suggestion
      <a href="#home">
        <img
          src="images/logo.png"
          alt="moyo logo"
          class="image"
        >
      </a>
```